refinery/user_files_manager/tests.py

```diff
@@ -41,7 +44,7 @@ def test_get(self):
 
 
 class UserFilesUITests(StaticLiveServerTestCase):
```

Comment:
Could inherit from `RefinerySeleniumTestBase` here

Reply:
If we're just doing HTTP and don't need a mock browser, is there are reason to use `RefinerySeleniumTestBase`?